### ISSUE TITLE
Create rom shortcut and pin the rom on the launcher

### ISSUE THREAD

**Issue by shabbirmahmud** [rafaelvcaetano/melonDS-android#267]
1-It will be great if I can open the rom directly from the phone launcher.

2-It will also be good if I can pin my favorite games on the top of the emulator launcher.

**Comment by rafaelvcaetano** [comment]
1. This is already implemented 😅 You just need to long press on your home screen, search for melonDS and the ROM Shortcut option should be available.
2. You can already sort by recently played. It's the same as favourites but it might me close enough. I will consider this anyway.

**Comment by shabbirmahmud** [comment]
> 1. This is already implemented 😅 You just need to long press on your home screen, select Widgets, search for melonDS and the ROM Shortcut option should be available.

Lol 😂😂.I didn't know.

**Comment by shabbirmahmud** [comment]
I checked in my widgets i didn't find any widgets in my realme 5i.

https://user-images.githubusercontent.com/47573399/113224746-e0983100-92ad-11eb-8ebc-20748ac06604.mp4

But i found in my redmi 5+.

**Comment by rafaelvcaetano** [comment]
> Isn't it possible it create like this for every rom?
> 
> ![Screenshot_2021-04-01-05-51-27-54](https://user-images.githubusercontent.com/47573399/113225013-7a5fde00-92ae-11eb-99b2-e1e7204cc51d.jpg)

Yes it is and it's already in my TODO list.

**Comment by shabbirmahmud** [comment]
> Yes it is and it's already in my TODO list.

Then should i close this issu?

**Comment by rafaelvcaetano** [comment]
No. Leave it open. Some of the other issues you opened were also already in my TODO list but this gives visibility to other users

Regarding point 2, what are your suggestions? Adding a star to mark ROMs as favourites and a sorting order to put favourite ROMs at the top? Or display favourite ROMs in a separate list? Neither of these options seem perfect to me so you can also suggest other alternatives.

**Comment by shabbirmahmud** [comment]
> No. Leave it open. Some of the other issues you opened were also already in my TODO list but this gives visibility to other users
> 

Ok.👍

> Regarding point 2, what are your suggestions? Adding a star to mark ROMs as favourites and a sorting order to put favourite ROMs at the top? Or display favourite ROMs in a separate list? Neither of these options seem perfect to me so you can also suggest other alternatives.

I was saying that if i pin any rom it will never move even i use other roms.Like this-

![Screenshot_2021-04-01-06-21-04-87](https://user-images.githubusercontent.com/47573399/113226782-d0368500-92b2-11eb-8a6b-4c57ecf92ec6.jpg)

I will recommend the last used rom will come top and it will be bottom of the pin rom.I think don't need to create a seperate favorite list.Star mark or ❣️ mark will do.

**Comment by shabbirmahmud** [comment]
You can also add rom sort like this-

https://user-images.githubusercontent.com/47573399/113227578-d0378480-92b4-11eb-9ae0-9aaf266b9419.mp4

**Comment by shabbirmahmud** [comment]
You can use Recent and All tabs rather than using last rom will come top like this-

![Screenshot_2021-04-01-06-49-43-15](https://user-images.githubusercontent.com/47573399/113228540-e9d9cb80-92b6-11eb-9072-247b1e7f411f.jpg)

**Comment by Gengar676** [comment]
Hello I'm also not getting the option for rom shortcut in my Widgets section

**Comment by Viriko23** [comment]
Wow it's been 3 years, my Redmi Note 11 doesn't have it either, on both the release on playstore and the nightly release
